dot/parachain/types/async_backing.go

```diff
@@ -17,3 +22,90 @@ type AsyncBackingParams struct {
 	// When async backing is disabled, the only valid value is 0.
 	AllowedAncestryLen uint32 `scale:"2"`
 }
+
+// InboundHrmpLimitations constraints on inbound HRMP channels.
+type InboundHrmpLimitations struct {
```

Comment:
Do we use this and the below code in any subsystem other than the prospective parachain subsystem?

Reply:
Currently we only use in prospective-parachains pkg, however since this types are a representation of what the runtime uses to validate the constraints I think placing them here is the better approach, given that they are in the same level as `CandidateHash`, `PersistedValidationData`

Reply:
In this package, we keep all the types that are used in multiple subsystems.

CandidateHashm and PersistedValidationData are the types we use in multiple subsystems.
As I can see In Rust implementation, the `InboundHrmpLimitations` type is only being used in prospective parachain.
Please move this to prospective parachain package